docs/api-specs/reporting-api-v1.yaml

```
@@ -182,11 +215,167 @@ paths:
                 $ref: '#/components/schemas/ReportData'
         404:
           description: report data is not available yet (when either the calculation id is not valid or the calculation is still running)
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
         401:
           description: If the access token does not grant permission to this Report
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
+  # notification
+
+  /webhook:
+    get:
+      security:
+        - development:
+          - reporting_read
+      tags:
+        - notification
+      summary: Return list of existing Webhooks
+      responses:
+        200:
+          description: List of all available Webhooks the requester has created, empty array if no Webhooks are available
```

Do I understand correctly that we are filtering by user/client here?

```suggestion
      summary: Return list of existing Webhooks for your authorized client
      responses:
        200:
          description: List of all available Webhooks the requester has created, empty array if no Webhooks are available for the client authorized in the given access token
```

---

Yes. Otherwise, different clients can edit different webhooks. Only users with the appropriate rights should be able to access all webhooks.

---

Absolutely agree.